apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/signature/SignatureToolFactory.java

```diff
@@ -25,14 +26,22 @@
 @NoArgsConstructor(access = AccessLevel.PACKAGE)
 public class SignatureToolFactory {
 
+    //TODO: read from properties, example: "elliptic-curve=Curve25519", "pk-size=32"
+    private static final List<String> DEFAULT_MULTISIG_V3_PARAMS = List.of();
```

Comment:
V3 sig verification is also not introduced, please revert the code or comment it

Reply:
done